chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/impl/CertificationCriterionDAOImpl.java

```diff
@@ -175,11 +176,22 @@ private void update(CertificationCriterionEntity entity) {
 		
 	}
 	
+	@Transactional
 	private List<CertificationCriterionEntity> getAllEntities() {
+		Query query = entityManager.createQuery(
+				"SELECT cce "
+				+ "FROM CertificationCriterionEntity cce "
+				+ "WHERE cce.id IN ( "
+				+ "SELECT c.id FROM CertificationCriterionEntity c "
+				+ "LEFT JOIN c.certificationResult cr "
+				+ "LEFT JOIN cr.certifiedProduct cp "
+				+ "LEFT JOIN cr.certificationResultTestTool crtt "
+				+ "LEFT JOIN crtt.testTool tt "
+				+ "WHERE (tt.retired = true AND cp.icsCode = '1') OR (tt.retired = false) AND c.deleted = false)"

```

Comment:
I think you might want "icsCode >= 1". ICS means "Inherited Certification Status" (i think?) and it is not a boolean but a count of the number of times something has been inherited... so it can be larger than 1.